src/Core/Checkout/Order/Event/OrderCriteriaEvent.php

```diff
@@ -0,0 +1,31 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Core\Checkout\Order\Event;
+
+use Shopware\Core\Framework\DataAbstractionLayer\Search\Criteria;
+use Shopware\Core\System\SalesChannel\SalesChannelContext;
+use Symfony\Contracts\EventDispatcher\Event;
+
+class OrderCriteriaEvent extends Event
```

Comment:
Just as icing on the cake, could you implement `\Shopware\Core\Framework\Event\ShopwareSalesChannelEvent` here? Then we will import it 👍

Reply:
I just added.